src/board.ts

```diff
@@ -62,8 +66,30 @@ export default class Board {
   // Sets object at (x, y).
   public setObj(x: number, y: number, obj: Tile): void {
     this.map[x][y] = obj;
-    this.context.fillStyle = this.game.getColour(obj.colour);
-    this.context.fillRect(x, y, 1, 1);
+
+    const rgb = this.getRGBColour(obj.colour);
```

Comment:
You could replace all the colors with tuple `[int, int, int]` and add
```
getRGB = (r, g, b) => `rgb(${r}, ${g}, ${b})`
```
, but this solution is ok too.
There is no reason to use hex at all, I just thought that would be best when I started writing it. Now I would probably use hsl because it's more human-readable.

Reply:
That's probably a good idea. The regex was an attempt to change as little as possible (I noticed the hex colors were used in at least one other place, so I didn't want to risk breaking anything). I'm not super happy with having to parse the hex values, especially since it needs a cache to be fast.

Reply:
I created #9 for this one. Maybe someone will do this